Author: Andrew Lee (李健秋) <ajqlee@debian.org>
Description: Support switch user with gdm3.

diff -Naurp lxsession.orig/lxsession-logout/lxsession-logout.c lxsession/lxsession-logout/lxsession-logout.c
--- lxsession.orig/lxsession-logout/lxsession-logout.c	2011-07-27 19:26:36.000000000 +0000
+++ lxsession/lxsession-logout/lxsession-logout.c	2012-04-15 11:21:14.746572092 +0000
@@ -442,6 +442,13 @@ int main(int argc, char * argv[])
         handler_context.switch_user_GDM = TRUE;
     }
 
+    /* If we are under GDM3, its "Switch User" is available. */
+    if (verify_running("gdm3", "gdmflexiserver"))
+    {
+        handler_context.switch_user_available = TRUE;
+        handler_context.switch_user_GDM = TRUE;
+    }
+
     /* If we are under KDM, its "Switch User" is available. */
     if (verify_running("kdm", "kdmctl"))
     {
