Display shutdown and reboot buttons for LTSP clients.

Index: lxsession-0.4.4/lxsession-logout/lxsession-logout.c
===================================================================
--- lxsession-0.4.4.orig/lxsession-logout/lxsession-logout.c	2010-09-10 14:07:12.000000000 -0700
+++ lxsession-0.4.4/lxsession-logout/lxsession-logout.c	2010-09-10 14:08:04.000000000 -0700
@@ -481,7 +481,11 @@
 
     /* LTSP support */
     if (g_getenv("LTSP_CLIENT"))
+    {
         handler_context.ltsp = TRUE;
+        handler_context.shutdown_available = TRUE;
+        handler_context.reboot_available = TRUE;
+    }
 
     /* Make the button images accessible. */
     gtk_icon_theme_append_search_path(gtk_icon_theme_get_default(), PACKAGE_DATA_DIR "/lxsession/images");
